app/compose.yaml

```diff
@@ -97,6 +97,31 @@ services:
     depends_on:
       - dev
 
+  postgresql:
+    container_name: groc-db
+    environment:
+      DATABASE_HOST: 127.0.0.1
+      POSTGRES_PASSWORD: groc
+      POSTGRES_USER: groc
+      POSTGRES_DB: groc
+    ports:
+      - 5432:5432
```

Comment:
Can you add this here please? This is nice to have to know if it suddenly goes down.

```
    restart: always
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -d groc -U groc"]
      interval: 1s
      timeout: 5s
      retries: 10
```